internal/output/plain.go

```diff
@@ -320,6 +320,21 @@ func sprintTable(slice []interface{}) (string, error) {
 	// Don't print whitespace lines
 	t.SetHideLines([]string{"betweenLine", "top", "aboveTitle", "LineTop", "LineBottom", "bottomLine"})
 	t.SetAlign("left")
+
+	// Set our own custom table format
+	tabulate.TableFormats["standard"] = tabulate.TableFormat{
+		LineTop:         tabulate.Line{"", "-", "", ""},
+		LineBelowHeader: tabulate.Line{"", "\u2500", "", ""},
+		LineBottom:      tabulate.Line{"", "-", "", ""},
+		HeaderRow:       tabulate.Row{"[BOLD]", "", "[/RESET]"},
```

Comment:
```suggestion
		HeaderRow:       tabulate.Row{"[HEADING]", "", "[/RESET]"},
```

Martin JUST pushed the change that removes `BOLD`.